Doc/library/sqlite3.rst

```diff
@@ -521,6 +521,34 @@ Connection Objects
                  f.write('%s\n' % line)
 
 
+   .. method:: backup(filename[, pages, progress])
+
+      This method exposes the `API`__ that allows to make a backup of a SQLite
+      database into the mandatory argument *filename*, even while it's being accessed
+      by other clients, or concurrently by the same connection.
+
+      By default, or when *pages* is either ``0`` or a negative integer, the entire
```

Comment:
Do you mean that I should remove the pages argument? But if I do that, also the progress one becomes pointless, because with the default value there will be at most one callback, when the engine completes the backup. What am I missing?